client/Wayland/wlfreerdp.c

```diff
@@ -74,10 +74,19 @@ static BOOL wl_end_paint(rdpContext* context)
 	display = context_w->display;
 	window = context_w->window;
 
-	for (i = 0; i < h; i++)
+	for (i = 0; i < h; i++) {
```

Comment:
Please note that this coding style isn't common for FreeRDP. The bracket should be on the next line.